### PR TITLE
fix: rename /api-clients/access-control-systems -> /api-clients/acs

### LINKED ISSUE #303: Create redirects for every page under `api-clients`
For every rename we do in https://github.com/seamapi/docs/issues/304, we need to add a redirect to https://github.com/seamapi/docs/blob/beta/.gitbook.yaml

For example, if we do

```diff
- * [Access Control Systems](api-clients/access-control-systems/README.md)
+ * [Access Control Systems](api-clients/acs/README.md)
-  * [Systems](api-clients/access-control-systems/systems/README.md)
+  * [Systems](api-clients/acs/systems/README.md)
-    * [List Systems](api-clients/access-control-systems/systems/list-systems.md)
+    * [List Systems](api-clients/acs/systems/list.md)
```

Then we need to add

```yaml
redirects:
  api-clients/access-control-systems: api-clients/acs/README.md
  api-clients/access-control-systems/systems: api-clients/acs/systems/README.md
  api-clients/access-control-systems/systems/list-systems: api-clients/acs/systems/list.md
```

We can then find/replace all links in the docs to point to the new files.

### LINKED ISSUE #304: Update API reference section to be under `/api` but link to files under `api-clients`
We need to turn every item under `📋 API Reference <a href="#api-clients" id="api-clients"></a>` in 
[SUMMARY.md](https://github.com/seamapi/docs/blob/beta/docs/SUMMARY.md) to match the final structure, and we need to rename the files to match.

For example,

```diff
- * [Access Control Systems](api-clients/access-control-systems/README.md)
+ * [Access Control Systems](api-clients/acs/README.md)
-  * [Systems](api-clients/access-control-systems/systems/README.md)
+  * [Systems](api-clients/acs/systems/README.md)
-    * [List Systems](api-clients/access-control-systems/systems/list-systems.md)
+    * [List Systems](api-clients/acs/systems/list.md)
```

### PR DESCRIPTION
**Pull request by mikewuu**
closes #303 
closes #304

Note: did **not** rename to remove resource.

eg. in the examples, this was one of the redirects

```diff
-    * [List Systems](api-clients/access-control-systems/systems/list-systems.md)
+    * [List Systems](api-clients/acs/systems/list.md)
```

But I've left it as the following for consistency with the rest of the files (not jus acs)

```diff
-    * [List Systems](api-clients/access-control-systems/systems/list-systems.md)
+    * [List Systems](api-clients/acs/systems/list-systems.md)
```

### CODE REVIEW

File: .gitbook.yaml [hunk 1]
```diff
@@ -31,3 +31,36 @@ redirects:
   device-and-system-integration-guides/assa-abloy-visionline-access-control-system-in-development/special-requirements-for-ios-mobile-access-sdk-development: device-and-system-integration-guides/assa-abloy-visionline-access-control-system/special-requirements-for-ios-mobile-access-sdk-development.md
   capability-guides/access-systems/understanding-acs-differences: capability-guides/access-systems/understanding-access-control-system-differences.md
   device-guides/get-started-with-schlage-locks: device-and-system-integration-guides/schlage-locks/get-started-with-schlage-locks.md
+  api-clients/access-control-systems: api-clients/acs/README.md
+  api-clients/access-control-systems/systems: api-clients/acs/systems/README.md
+  api-clients/access-control-systems/systems/get-system: api-clients/acs/systems/get-system.md
```
Comment (razor-x) [inline .gitbook.yaml:36]:
```suggestion
  api-clients/access-control-systems/systems/get-system: api-clients/acs/systems/get.md
```

Etc.

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -163,40 +163,40 @@
...
-    * [Get a Credential](api-clients/access-control-systems/credentials/get-credential.md)
-    * [Update a Credential](api-clients/access-control-systems/credentials/update-a-credential.md)
-    * [Unassign a Credential from a User](api-clients/access-control-systems/credentials/unassign-a-credential-from-a-user.md)
-    * [Delete a Credential](api-clients/access-control-systems/credentials/delete-credential.md)
-    * [List Accessible Entrances](api-clients/access-control-systems/credentials/list-accessible-entrances.md)
+    * [Get a System](api-clients/acs/systems/get-system.md)
+    * [List Compatible Credential Manager ACS Systems](api-clients/acs/systems/list-compatible-credential-manager-acs-systems.md)
+  * [Users](api-clients/acs/users/README.md)
+    * [Create a User](api-clients/acs/users/create-user.md)
+    * [List Users](api-clients/acs/users/list-users.md)
```
Comment (razor-x) [inline docs/SUMMARY.md:173]:
```suggestion
    * [List Users](api-clients/acs/users/list.md)
```
Etc.

**Review by razor-x — CHANGES_REQUESTED** [review]
We need to do this for every page under the api-clients folder (not just ACS). This is because of how gitbook enforces the path matching.

**Comment by mikewuu** [comment]
@razor-x renamed everything in **/api-clients**. Wasn't 100% on the naming for some of the routes, specifically ones that include other resource names, like **list-credentials-with-access-to-an-entrance**.

See https://github.com/seamapi/docs/pull/308/files#diff-3bdea6261687f749bccd68763ce01cfe6249305137f746ef6ef7c3f7eadebd0d for all the renamings

File: docs/SUMMARY.md (hunk 2 above)
Comment (mikewuu) [reply docs/SUMMARY.md:173]: updated in https://github.com/seamapi/docs/pull/308/commits/b01416b3487f42a305527833db477602501fe6ed

**Comment by razor-x** [comment]
@mikewuu All the routes need to match the API exactly, i.e., every file is 1to1 with and API route. 


```
api-clients/acs/entrances/list-credentials-with-access-to-an-entrance.md => api-clients/acs/entrances/list_credentials_with_access.md
```

Additionally, we need all the paths to match the API, so for example, these still need to be fixed, etc.


```
docs/api-clients/devices/list-unmanaged.md => docs/api-clients/devices/unmanaged/list.md
docs/api-clients/user-identities/add-an-acs-user.md => docs/api-clients/user_identities/add_acs_user.md
... any file that does not match the API route it documents exactly
```

**Comment by mikewuu** [comment]
@razor-x , closing in favor of #312  as this PR got stale.

**Pull request closed without merging**